client/src/styles/app.css

```diff
@@ -0,0 +1,5 @@
+body{
+  background-color: #395155;
+  color: #F4F3F3;
+  font-family: 'Play', sans-serif;
+}
```

Comment:
You'll need to import this file into the react app for it to have any effect. Also we should already have the correct colors and fonts loaded into our bootstrap css via sass variables, check out `styles/bootstrap-theme/wepickheroes.css`.